src/app/application.h

```diff
@@ -132,7 +132,7 @@ private slots:
 #endif
 
 #ifndef DISABLE_WEBUI
-    QPointer<WebUI> m_webui;
+    WebUI *m_webui;
```

Comment:
You forgot to pass parent to the constructor.

Reply:
No. We delete Application components manually since the ordered cleanup is required.
But now we have to do the cleanup also in the case of a failed Application launch.

Reply:
Then `parent` parameter for the `WebUI` constructor is useless.

Reply:
Ok. Removed.